packages/inventory-compliance/src/SystemRulesTable.js

```diff
@@ -342,7 +342,7 @@ class SystemRulesTable extends React.Component {
     compressRows = (rows) => {
         const compressedRows = [];
         rows.forEach((row, i) => {
-            if (row.hasOwnProperty('isOpen')) {
+            if (this.isParent(row)) {
```

Comment:
There's a function for `row.hasOwnProperty('isOpen')` that's more meaningful - I'm just using it, there's no actual change in behavior here